eclass/dist-kernel-utils.eclass

```diff
@@ -175,7 +175,8 @@ dist-kernel_get_module_suffix() {
 		echo .ko
 	elif [[ ! -r ${config} ]]; then
 		die "Cannot find kernel config ${config}"
-	elif grep -q "CONFIG_MODULE_COMPRESS_NONE=y" "${config}"; then
+	elif grep -q "CONFIG_MODULE_COMPRESS_NONE=y" "${config}" ||
```

Comment:
The change is good, I think, but I'll note that in part, the only reason I noticed the problem was because of this check failing. But nothing we can do about that.